rfcs/0000-rfc-session.md

```diff
@@ -0,0 +1,202 @@
+-->
+
+## Usage
+
+Session fields are used to describe the sesison attributes of:
+ - Cleint VPN Sessions
+ - Network to Network VPN Sessions
+ - Network Access Sessions (NAC, WPA, EAP, etc.)
+ - Local or remote device login sessions (RDP, ICA, xWindows)
+ - administrative sessions on infrastructure devices
+ - administrative sessions on cloud or application management portals
+ - applications sessions (e.g. sql server odbc session, application acces session)
+ 
+
+## Source data
+Source data expectations include:
+ - Wireless Lan Controllers
+ - Security appliances
+ - Network admission control devices
+ - Radius / tacacs servers
+ - Application server logs
+
+Example 1: Meraki 802.1x Logs (WLC)  (EAP session start)
+<134>1 1580551704.928047208 my_AP events type=8021x_eap_success radio='1' vap='2' client_mac='12:34:56:78:9A:BC' client_ip='192.168.1.100' identity='JohnDoe' aid='1687088497’
+
+802.1x EAP De-association Message  (EAP session end)
+<134>1 1580551705.928047208 my_AP events type=8021x_deauth radio='1' vap='2' identity='JohnDoe' aid='1687088497’'
+
+ - Note, while there is an association id (session.id) created prior to wpa/802.1x authentication, building the session event from the eap success message allows for easier integration
+ of fields like username, client.ip, etc. in an 802.1x or WPA environment
+
+  - - Base 802.11 Association:  (802.11 session start)
+  - - - <134>1 1380653443.857790533 MR18 events type=association radio='1' vap='1' channel='2' rssi='23' aid='1687088497’
+
+  - - Base 802.11 Deassociation Message  (802.11 session end)
+  - - - 1380653443.857790533 my_AP events type=disassociation radio='1' vap='2' channel='6' reason='8' instigator='2' duration='11979.728000' auth_neg_dur='1380653443.85779053324000' last_auth_ago='5.074000' is_wpa='1' full_conn='1.597000' ip_resp='1.597000' ip_src='192.168.111.251' arp_resp='1.265000' arp_src='192.168.111.251' dns_server='192.168.111.1' dns_req_rtt='1380653443.85779053335000' dns_resp='1.316000' aid='1813578850'
+
+
+Example 2: ASA Admin Login
+ - Session start
+  - - <166>Feb 03 2020 11:27:05 5508x-1_9.12(3): %ASA-6-605005: Login permitted from 192.168.1.250/59277 to management:192.168.1.10/ssh for user "JohnDoe"
+- Session End
+ - - <166>Feb 03 2020 11:27:05 5508x-1_9.12(3): %ASA-6-315011: SSH session from 192.168.1.250 on interface management for user JohnDoe disconnected by SSH server, reason: timeout
+
+Example 3: ASA Web VPN
+- Session Start
+- - <166>Feb 03 2020 11:27:05 5508x-1_9.12(3): %ASA-6-721016: WebVPN session for client user JohnDoe , 192.168.1.100 has been created.
+
+- Session End:
+- - <166>Feb 03 2020 11:27:05 5508x-1_9.12(3):%ASA-6-721018: WebVPN session for client user JohnDoe , IP 192.168.1.100 has been deleted.
```

Comment:
```suggestion
* Note, while there is an association id (session.id) created prior to wpa/802.1x authentication, building the session event from the eap success message allows for easier integration
 of fields like username, client.ip, etc. in an 802.1x or WPA environment

    * Base 802.11 Association:  (802.11 session start)
        * <134>1 1380653443.857790533 MR18 events type=association radio='1' vap='1' channel='2' rssi='23' aid='1687088497’

    * Base 802.11 Deassociation Message  (802.11 session end)
        * 1380653443.857790533 my_AP events type=disassociation radio='1' vap='2' channel='6' reason='8' instigator='2' duration='11979.728000' auth_neg_dur='1380653443.85779053324000' last_auth_ago='5.074000' is_wpa='1' full_conn='1.597000' ip_resp='1.597000' ip_src='192.168.111.251' arp_resp='1.265000' arp_src='192.168.111.251' dns_server='192.168.111.1' dns_req_rtt='1380653443.85779053335000' dns_resp='1.316000' aid='1813578850'


Example 2: ASA Admin Login
* Session start
    * <166>Feb 03 2020 11:27:05 5508x-1_9.12(3): %ASA-6-605005: Login permitted from 192.168.1.250/59277 to management:192.168.1.10/ssh for user "JohnDoe"
* Session End
    * <166>Feb 03 2020 11:27:05 5508x-1_9.12(3): %ASA-6-315011: SSH session from 192.168.1.250 on interface management for user JohnDoe disconnected by SSH server, reason: timeout

Example 3: ASA Web VPN
* Session Start
    * <166>Feb 03 2020 11:27:05 5508x-1_9.12(3): %ASA-6-721016: WebVPN session for client user JohnDoe , 192.168.1.100 has been created.

* Session End:
    * <166>Feb 03 2020 11:27:05 5508x-1_9.12(3):%ASA-6-721018: WebVPN session for client user JohnDoe , IP 192.168.1.100 has been deleted.
```

Markdown is being markdown and doing strange things with these bullets. I tried clearing it up a bit.